vehicle/raw_vehicle_cmd_converter/src/node.cpp

```diff
@@ -86,8 +86,8 @@ RawVehicleCommandConverterNode::RawVehicleCommandConverterNode(
   pub_actuation_cmd_ = create_publisher<ActuationCommandStamped>("/vehicle/actuation_cmd", 1);
   sub_control_cmd_ = create_subscription<ControlCommandStamped>(
     "/control/control_cmd", 1, std::bind(&RawVehicleCommandConverterNode::onControlCmd, this, _1));
-  sub_velocity_ = create_subscription<TwistStamped>(
-    "/localization/twist", 1, std::bind(&RawVehicleCommandConverterNode::onVelocity, this, _1));
+  sub_velocity_ = create_subscription<Odometry>(
+    "/localization/odometry", 1, std::bind(&RawVehicleCommandConverterNode::onVelocity, this, _1));
```

Comment:
Topic for odometory may be /localization/kinematic_state or something?

Reply:
```
/control/command/ackermann_control_command
/localization/kinematic_state
/parameter_events
/rosout
/vehicle/command/actuation_command
/vehicle/raw_vehicle_cmd_converter/debug/steer_pid
/vehicle/status/steering

```

Reply:
Ah you decided the topic name /control/command/ackermann_control_command!